microsoft-edge/devtools-guide.md

```diff
@@ -14,62 +14,66 @@ experiment_id: "51fe4b97-3e55-41"
+
+For more on the latest features, check out [*DevTools in the latest update of Windows 10 (EdgeHTML 18)*][DevtoolsGuideEdgehtmlWhatsnew].  
 
 ## Core tools  
 
-![Microsoft Edge \(EdgeHTML\) DevTools](./devtools-guide/media/devtools.png)  
+![Microsoft Edge \(EdgeHTML\) DevTools][ImageDevtoolsEdgehtml]  
 
 The Microsoft Edge \(EdgeHTML\) DevTools include:  
 
-*   An [**Elements**](./devtools-guide/elements.md) panel to edit HTML and CSS, inspect accessibility properties, view event listeners, and set DOM mutation breakpoints  
-*   A [**Console**](./devtools-guide/console.md) to view and filter log messages, inspect JavaScript objects and DOM nodes, and run JavaScript in the context of the selected window or frame  
-*   A [**Debugger**](./devtools-guide/debugger.md) to step through code, set watches and breakpoints, live edit your code, and inspect your web storage and cookie caches  
-*   A [**Network**](./devtools-guide/network.md) panel to monitor and inspect requests and responses from the network and browser cache  
-*   A [**Performance**](./devtools-guide/performance.md) panel to profile the time and system resources required by your site  
-*   A [**Memory**](./devtools-guide/memory.md) panel to measure your use of memory resources and compare heap snapshots at different states of code execution  
-*   A [**Storage**](./devtools-guide/storage.md) panel for inspecting and managing your web storage, IndexedDB, cookies and cache data  
-*   A [**Service Workers**](./devtools-guide/service-workers.md) panel for managing and debugging your service workers  
-*   An [**Emulation**](./devtools-guide/emulation.md) panel to test your site with different browser profiles, screen resolutions, and GPS location coordinates  
+*   An [**Elements**][DevtoolsGuideEdgehtmlElements] panel to edit HTML and CSS, inspect accessibility properties, view event listeners, and set DOM mutation breakpoints  
+*   A [**Console**][DevtoolsGuideEdgehtmlConsole] to view and filter log messages, inspect JavaScript objects and DOM nodes, and run JavaScript in the context of the selected window or frame  
+*   A [**Debugger**][DevtoolsGuideEdgehtmlDebugger] to step through code, set watches and breakpoints, live edit your code, and inspect your web storage and cookie caches  
+*   A [**Network**][DevtoolsGuideEdgehtmlNetwork] panel to monitor and inspect requests and responses from the network and browser cache  
+*   A [**Performance**][DevtoolsGuideEdgehtmlPerformance] panel to profile the time and system resources required by your site  
+*   A [**Memory**][DevtoolsGuideEdgehtmlMemory] panel to measure your use of memory resources and compare heap snapshots at different states of code runtime  
+*   A [**Storage**][DevtoolsGuideEdgehtmlStorage] panel for inspecting and managing your web storage, IndexedDB, cookies and cache data  
```

Comment:
Links are broken on the review site. Expected?

Reply:
Updated. 
The links that contained `.md` were the culprits.